flink-contrib/flink-storm/src/main/java/org/apache/flink/storm/api/FlinkLocalCluster.java

```diff
@@ -99,6 +111,7 @@ public void rebalance(final String name, final RebalanceOptions options) {
 
 	public void shutdown() {
 		flink.stop();
+		flink = null;
```

Comment:
Should be kept. Otherwise, calling `submitTopologyWithOpts` a second time will run into a NPE. (Or add proper exception as `else` of `if (flink == null)` check, ie, "Cannot run topology. Cluster got shut down." or similar.)